projects/projectsRouter.js

```diff
@@ -0,0 +1,104 @@
+const express = require("express");
+const projectDb = require("../data/helpers/projectModel");
+const router = express.Router();
+
+router.get("/", (req, res) => {
+    projectDb.get()
+    .then(post => {
+        res.status(200).json(post)
+    })
+    .catch(err => {
+        res.status(400).send("Error no data returned")
+    })
+})
+
+router.get("/:id", validateActionId, (req, res) => {
+    projectDb.get(req.params.id)
+    .then(proj => {
+        res.status(200).json(proj)
+    })
+    .catch(err => {
+        res.status(400).json({
+            error : "Did not retrieve id"
+        })
+    })
+})
+
+router.post("/", (req, res) => {
+    projectDb.insert(req.body)
+    .then(proj => {
+        res.status(200).json(proj)
+    })
+    .catch(err => {
+        res.status(400).json({
+            error : "Trouble posting project..."
+        })
+    })
+})
+
+router.post("/:id/actions", validateActionId, (req, res) => {
+    projectDb.insert(req.body)
+    .then(post => {
```

Comment:
The action could use another validator to name sure the required `description` and `notes` are present